### ISSUE TITLE
How to use this project with other libraries ?

### ISSUE THREAD

**Issue by kracekumar** [replit-archive/empythoned#11]
Can you explain the process how the this code was generated? Is Cpython code converted using emscripten? if so can you please explain the steps.

In order to use empythoned in my project, is it enough to load `dist/worker.js` in my html and which will load `dist/python.opt.js`?

**Comment by kracekumar** [comment]
I am able to use js in `dist`, here is the demo `https://github.com/kracekumar/test-empythoned`, just wanted to know how to convert `CPython to js`.

**Comment by kracekumar** [comment]
@nightpool: I got that, but I am looking for converting CPython to JS. `emcc` takes single file, looking how to convert entire code base.

**Comment by nightpool** [comment]
Use the `build` script in the main directory? 

EDIT: Oops, I just deleted my last comment. Here's what was in it, for posterity:
What empythoned converts is the CPython interpreter. You can then use that to run (in the browser) arbitrary ~~javascript~~ (ED: typo) python code.

**Comment by kracekumar** [comment]
Thanks @nightpool .

**Comment by kracekumar** [comment]
Seems issue with empythoned build script. https://github.com/kripken/emscripten/issues/1011

**Comment by astromme** [comment]
I am having the same problem that kracekumar is having when I try and use empythoned from a chrome app.

**Comment by kracekumar** [comment]
@astromme I was able to create chrome extension [offline-python](https://github.com/kracekumar/offline-python), but `import` statement fails.

**Comment by astromme** [comment]
Yes, thats what I mean. I did a little bit of digging, and I think there is some bug in how chrome packaged apps and extensions handle the emscripten FS.
